ods_ci/tests/Resources/CLI/AcceleratorProfiles/AcceleratorProfiles.resource

```diff
@@ -0,0 +1,100 @@
+*** Settings ***
+Documentation    Collection of keywords to interact with the administration of Accelerator Profiles via CLI
+
+
+*** Variables ***
+${ACCELERATOR_RESOURCES_DIRPATH}=    ods_ci/tests/Resources/Files/accelerator
+
+
+*** Keywords ***
+Create An Accelerator Profile Via CLI
+    [Documentation]    Create an instance of Accelerator Profile using OC
+    [Arguments]   ${name}  ${ns}=redhat-ods-applications
```

Comment:
Instead of redhat-ods-applications it should be `${APPLICATIONS_NAMESPACE}` to make it compatible with odh-nightly. Same comment for other places in the same PR

Reply:
Done [45138df](https://github.com/red-hat-data-services/ods-ci/pull/1209/commits/45138df54bbb0bd0091701d89f9da46341d8b3c6)****